src/internal/authdb/authdb.go

```diff
@@ -12,11 +14,12 @@ const (
 	roleBindingsCollectionName = "role_bindings"
 	membersCollectionName      = "members"
 	groupsCollectionName       = "groups"
+	InternalUser               = auth.InternalPrefix + "auth-server"
```

Comment:
For this & all the other exported items, be sure to add some documentation which would be useful when using godoc.